simulation/vis_simulate.pro

```diff
@@ -157,6 +159,16 @@ FUNCTION vis_simulate,obs,status_str,psf,params,jones,skymodel,file_path_fhd=fil
       
     endif ;; end if n_elements(model_image_cube) gt 0 or n_elements(model_uvf_cube) gt 0 or keyword_set(include_eor)
     
+    ;; If bubble file is included, add to the model_uvf_arr
+    IF Keyword_Set(bubble_fname) THEN BEGIN 
+       bubble_uvf = eor_bubble_sim(obs, jones, select_radius=select_radius, bubble_fname=bubble_fname)
+       IF Min(Ptr_valid(bubble_uvf)) GT 0 THEN BEGIN
+         IF n_elements(model_uvf_arr) GT 0 then begin
+           FOR pol_i=0,n_pol-1 DO *model_uvf_arr[pol_i] += *bubble_uvf[pol_i];*uv_mask_use
+         endif else model_uvf_arr = Pointer_copy(bubble_uvf)
```

Comment:
Please make sure that the bubble_uvf is in the right units. I've had trouble in the past (and still do!) about whether or not we are using XX+YY/2 or XX+YY. I don't know what the right answer is, but we should definitely look into it. We should compare directly to another simulation and see if we're off by a factor of 2...

Reply:
By my reading of stokes_cnv.pro, the convention in FHD seems to be the following:
``` I = XX + YY
XX = (I + Q)/2
YY = (I - Q)/2 
```
The bubble_uvf array obtained from eor_bubble_sim is in the (xx,yy,xy,yx) basis, having been converted with stokes_cnv(.... ,/inverse). It should therefore obey the same Stokes convention as the rest of FHD.